### PR TITLE
UX improvements for Merge configuration

### PR DESCRIPTION
**Pull request by MarekM25**
UX improvements for merge configuration + log fix

### CODE REVIEW

**Review by asdacap — COMMENTED** [review]
LGTM

**Review by asdacap — APPROVED** [review]

File: src/Nethermind/Nethermind.Merge.Plugin/MergePlugin.cs [hunk 1]
```diff
@@ -120,7 +131,8 @@ private void EnsureReceiptAvailable()
             {
                 if (!syncConfig.DownloadReceiptsInFastSync || !syncConfig.DownloadBodiesInFastSync)
                 {
-                    throw new InvalidOperationException("Receipt and body must be available for merge to function");
+                    if (_logger.IsError) _logger.Error("Receipt and body must be available for merge to function. The following configs values should be set to true: Sync.DownloadReceiptsInFastSync, Sync.DownloadBodiesInFastSync");
+                    _environment.Exit(1);
```
Comment (LukaszRozmej) [inline src/Nethermind/Nethermind.Merge.Plugin/MergePlugin.cs:135]: I would create some static ExitCodes class and name those + make unique codes

**Review by asdacap — APPROVED** [review]

File: src/Nethermind/Nethermind.Merge.Plugin/MergePlugin.cs (hunk 1 above)
Comment (MarekM25) [reply src/Nethermind/Nethermind.Merge.Plugin/MergePlugin.cs]: > I would create some static ExitCodes class and name those + make unique codes, so if we get info from users we can actually diagnose it easily

Done :)

File: src/Nethermind/Nethermind.Config/ExitCodes.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+//  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
+//
+
+namespace Nethermind.Config;
+
+public static class ExitCodes
+{
+    public const int NoEngineModule = 1;
+
+    public const int NoDownloadOldReceiptsOrBlocks = 2;
```
Comment (LukaszRozmej) [inline src/Nethermind/Nethermind.Config/ExitCodes.cs:24]: I would divide the errors in some numbers, for example configuration errors would be between 1_000 and 1_999, wouldn't go from 1 :)

**Review by marcindsobczak — APPROVED** [review]

**Review by LukaszRozmej — APPROVED** [review]

**Pull request merged**